soc/Kconfig

```diff
@@ -53,3 +53,9 @@ config SOC_HAS_TIMING_FUNCTIONS
 	help
 	  Should be selected if SoC provides custom method for retrieving
 	  timestamps and cycle count.
+
+config SOC_SUPPORTS_COREDUMP
```

Comment:
I don't think we need a new `SOC_SUPPORTS_*` for Xtensa. Just use the `ARCH_*` one as we know each SoC can be its own architecture for Xtensa.

Reply:
Changes made :+1: